src/core/ERC7779Adapter.sol

```diff
@@ -16,7 +25,21 @@ abstract contract ERC7779Adapter {
     storage at this slot, but just append their base to the array.
     *         This append operation should be done during the initialization of the account.
     */
-    function accountStorageBases() external view returns (bytes32[] memory) { }
+    function accountStorageBases() external view returns (bytes32[] memory) {
+        ERC7779Storage storage $;
+        assembly {
+            $.slot := ERC7779_STORAGE_BASE
+        }
+        return $.storageBases;
+    }
+
+    function _addStorageBase(bytes32 storageBase) internal {
```

Comment:
we will also need to add this: https://github.com/erc7579/erc7579-implementation/blob/c7e5a70a382b89c7a0c515c2224cdf880d56df0b/src/core/HookManager.sol#L20

Reply:
you mean not here, but at the initialization?